osh.spec

```diff
@@ -5,7 +5,9 @@ Version:        %{version}
 Release:        1%{?dist}
 License:        GPL-3.0-or-later
 Summary:        Static and Dynamic Analysis as a Service
-Source:         %{name}-%{version}.tar.bz2
+URL:            https://github.com/openscanhub/openscanhub/
+# TODO: This should be changed to releases once we start doing releases through GitHub
+Source:         https://github.com/openscanhub/openscanhub/archive/refs/tags/%{name}-%{version}.tar.bz2
```

Comment:
@kdudka Shall we create a release on GitHub from latest tag, so that we can use release url here?

Reply:
I did but it did not do what you had expected.  See #135 for details.